warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/MetadataTableSplits.java

```diff
@@ -184,6 +187,7 @@ public void update() {
                 log.info("updating splits file");
```

Comment:
Do you think we should update this log message to include the path?

Reply:
I didn't mess with it because the path will get logged within the call to createTempFile on 181.
`             Path tmpSplitsFile = createTempFile(fs);`